cmd/warpforge/catalog.go

```diff
@@ -59,32 +59,32 @@ var catalogCmdDef = cli.Command{
 		},
 		{
 			Name:   "ls",
-			Usage:  "List available catalogs",
+			Usage:  "List available catalogs in the root workspace",
 			Action: cmdCatalogLs,
 		},
 		{
 			Name:   "show",
-			Usage:  "Show the contents of a module",
+			Usage:  "Show the contents of a module in the root workspace catalog",
 			Action: cmdCatalogShow,
 		},
 		{
 			Name:   "bundle",
-			Usage:  "Bundle required catalog items into this project's catalog",
+			Usage:  "Bundle required catalog items into the current directory's workspace catalog. Will create a workspace and catalog in the current directory if none exists.",
```

Comment:
... I hope this isn't what it does?  Creating a workspace, that is?  If it doesn't find one it shouldn't make one.

I'd hope it's accurate to say it does this:

```suggestion
			Usage:  "Bundles any catalog information used by any modules in the current workspace into this workspace's catalog.  Use this to make a workspace independent.",
```

Reply:
Uhh well...
https://github.com/warpfork/warpforge/blob/4928d809fb1ed3cbe2e3e6a2d7f1e508a4c53c02/cmd/warpforge/catalog.go#L359-L362
Yeah I think that's what it does. For this PR I opted for describing _current_ behavior (AFAICT at least) over trying to make it the _desired_ behavior.

Reply:
Oh dear.  You're right.  And that's also definitely wrong, behaviorally.  Let's... fix that real quick, right after this >.>